components/gfx/platform/windows/font.rs

```diff
@@ -40,11 +42,205 @@ impl FontTableMethods for FontTable {
     }
 }
 
+fn make_tag(tag_bytes: &[u8]) -> FontTableTag {
+    assert_eq!(tag_bytes.len(), 4);
+    unsafe { *(tag_bytes.as_ptr() as *const FontTableTag) }
+}
+
+macro_rules! try_lossy(($result:expr) => (try!($result.map_err(|_| (())))));
+
+// Given a set of records, figure out the string indices for the family and face
+// names.  We want name_id 1 and 2, and we need to use platform_id == 1 and
+// language_id == 0 to avoid limitations in the truetype crate.  We *could* just
+// do our own parsing here, and use the offset/length data and pull the values out
+// ourselves.
+fn get_family_face_indices(records: &[truetype::naming_table::Record]) -> Option<(usize, usize)> {
+    let mut family_name_index = None;
+    let mut face_name_index = None;
+
+    for i in 0..records.len() {
+        // the truetype crate can only decode mac platform format names
```

Comment:
Should we support this? If so, consider filling an issue and referencing it here.